managing-packages/packages-and-managing-packages.md

```diff
@@ -109,58 +109,75 @@ To use `pip`, we should learn:
 1. How to list all the packages installed on our computer
 1. How to install packages
 
-### !callout-danger
-
-## Always use pip3
-
-In the following sections, our commands begin with `pip3`. That may be surprising; why do we use `pip3` instead of `pip`? `pip3` will specify to use `pip` with Python 3, which is not only desired, but **necessary.**
-
-### !end-callout
-
-<!-- Need to confirm that python3 install pip does not make this point moot -->
-
 ### Installation
 
-Starting with Python 3.4, `pip` gets installed during the Python installation!
+During "installfest" we used a tool named `pyenv` to install Python and `pip` along with it.
 
 To verify that `pip` is installed, we can run our first `pip` command in the terminal:
 
 ```bash
-pip3 --version
+pip --version
 ```
 
-If there was output that `pip3` is not found, refer to the `pip` docs for an installation guide.
+And we should see output similar to:
 
-### List All Packages
+```bash
+pip 24.3.1 from /Users/<YOUR_USER_NAME>/.pyenv/versions/3.13.1/lib/python3.13/site-packages/pip (python 3.13)
+```
 
-To list all of the packages and their version numbers installed on our computer, we can use `pip3 list`.
+When reading our output we should look out for:
+- If we see a messgae that `pip` is not found. 
+  - If you see this, refer to the `pip` docs for an installation guide.
+- If we see that `.pyenv` is not part of the path to the `pip` installation. 
+  - If you see this, reach out in #study-hall on Slack for assistance.
```

Comment:
Started with a small typo, but I also struggled with all the If/If/If/If here. What about rephrasing something like this?

```suggestion
- Do we see a message that `pip` is not found?
  - If we get a not found message, refer to the `pip` docs for an installation guide.
- Do we _not_ see `.pyenv` included in the path to the `pip` installation?
  - If the path doesn't include `.pyenv`, reach out in #study-hall on Slack for assistance.
```

Reply:
Also, is refering folks to a pip installation guide what we want to do? It should have come with `pyenv`, so if it's missing I'd like to know why, and make sure it's resolved appropriately. Maybe just send them to #study-hall in either situation?

Reply:
Yeah, that's the original advice, I agree that sending them to #study-hall would likely be preferred if pip is somehow not installed/in their path